src/pages/AddLiquidity/APRCalculation.tsx

```diff
@@ -23,7 +23,7 @@ export const apr = (
   let rewardPerBlock = adminRate
   const totalStakeBN = new BN(totalStake.toString())
   const totalSupplyUniBN = new BN(totalSupplyUni.toString())
-  const blocksPerWeek = 6000 * 7
+  const blocksPerWeek = 7200 * 7
```

Comment:
blocksPerWeek in the smart contract is still the same (6000 * 7).
I think what we need to change is how we calculate rewards per year, which should be rewardPerBlock * 7200 * 7 * 52

Reply:
It has already been calculated when getting APR.

Reply:
the thing is that if we change blocksPerWeek in line 26, it gonna change the way currEpoch is calculated (instead of being calculated as currentBlockNum.sub(endColBlock).div(6000*7), currEpoch is calculated as currentBlockNum.sub(endColBlock).div(7200*7), which is not correct)

Reply:
@cuongquangnam Check again, please.